plans/packit-integration.fmf

```diff
@@ -3,6 +3,10 @@ discover:
     url: https://github.com/packit/packit
     filter: tier:0 | tier:1
 
+prepare:
+    how: install
```

Comment:
Based on the error in EPEL, this has to be a list because of the adjust. In the adjust, we use `    prepare+:
`, note the plus - it expects lists to be merged. So hopefully

```
prepare:
  - how: install
    copr: ...
```

should remove the error...